XcodeServerSDK/Utils/JSON.swift

```diff
@@ -9,7 +9,7 @@
 import Foundation
 
 public protocol JSONSerializable {
-    init?(json: NSDictionary)
+    init?(json: NSDictionary) throws
```

Comment:
Wow, thanks for that link, I had no idea (or never actually thought about) passing a non-throwing function into a `throws` signed param. Makes sense and yeah that's what I like, that we can gradually migrate all the initializers over.